src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -308,53 +292,90 @@ export default class OrganizationDiscover extends React.Component {
               />
             </Fieldset>
           </Sidebar>
-          <Flex flex="1" direction="column" px={3} py={3}>
-            {data && (
-              <Result
-                data={data}
-                query={query}
-                chartData={chartData}
-                chartQuery={chartQuery}
+          <Flex direction="column" flex="1">
+            <ProjectAndTimeSelector>
+              <MultipleProjectSelector
+                value={currentQuery.projects}
+                projects={this.props.organization.projects}
+                onChange={val => this.updateField('projects', val)}
+                onUpdate={this.runQuery}
+                align="left"
+                labelMargin="-5px"
               />
-            )}
-            {!data && <Intro updateQuery={this.updateFields} />}
-            <EarlyAdopterMessage />
+              <HeaderSeparator />
+              <TimeRangeSelector
+                start={currentQuery.start}
+                end={currentQuery.end}
+                onChange={(name, val) => this.updateField(name, val)}
+                onUpdate={this.runQuery}
+                relative="14d"
```

Comment:
Lets not switch to relative time as part of this PR, there's a bunch of other changes that we'd need for this to work properly. For example the timestamps in URL won't work properly anymore

Reply:
👍